### ISSUE TITLE
[Bug] - Upon the "closed door" greeting, Monika doesn't actually fix the windows if you open the door.

### ISSUE THREAD

**Issue by chillandgrill** [Backdash/MonikaModDev#477]
I had to modify the code a little bit to test this, but I don't think it should've changed the script for Monika setting up the room. Anyway, if you open the door and head in, Monika sets up the room but forgets to fill in the windows. You remind her, and she's supposed to fill in the windows. During nighttime (haven't had a chance to test it in the day), Monika doesn't actually fix the windows. They remain black. As a side-note, when you disable and then enable room animation, the windows stay black after you re-enable unless you restart. This second tidbit isn't a huge issue, but could be a low-priority fix if possible.

**Comment by chillandgrill** [comment]
Oh yeah, and the music jumps. I'm not sure if that's a limitation of ren'py or because there's a redundant BG call somewhere, but if it's something fixable, I wanted to let you guys know about that too.

**Comment by ThePotatoGuy** [comment]
What do you mean by the music jumps? And where are you experiencing the jump?

**Comment by chillandgrill** [comment]
About the music, actually, I think it just sounded like a jump because I was using the exact same track as music the last time I had closed the game. So that’s not really an issue, actually, just a coincidence.

**Comment by chillandgrill** [comment]
For repro, I ended up deleting all the other greetings to ensure that I’d get the closed door one, but that’s about all the modification I did. Oh yeah, and renaming the label so that it would be a candidate for the random greeting, otherwise the game gave me an error that I didn’t investigate into. (I don’t have much experience with python in particular)

**Comment by ThePotatoGuy** [comment]
The greeting is actually **not** a candidate for the random greeting. It works via separate random chance that takes precdence over the regular random greeting. 

When I asked about workflow, I didn't mean testing flow. I meant any settings you had (like disabled animations) or keys pressed (like opening the `game_menu` during the greeting) that you did that led to the issue.

**Comment by chillandgrill** [comment]
Oh, okay. That makes sense. I read somewhere that it wasn't a regular random greeting. Explains why I had to change the label to guarantee I'd get it.

Sorry about the workflow thing, I'm not very good with development phrases. Animations were enabled, and I don't remember for sure but I'm quite certain that I didn't open the game menu during the greeting. IF I had pressed any keys or anything out of the normal, it would have been the menu, though.
